copy-pkg-json.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+const fs = require('fs');
+
+// get the package.json file
+const pkg = JSON.parse(fs.readFileSync('package.json'));
+
+// modify the "main" and "types" fields of the package.json file
+pkg.main = 'esm/index.js';
+pkg.types = 'dts/index.d.ts';
+
+// write the new package file to dist
+fs.writeFileSync('dist/package.json', JSON.stringify(pkg, null, 2));
```

Comment:
This is just a personal preference of mine, but could `copy-pkg-json.js` be put inside a new `scripts` directory that can be created/placed in the root directory, just to keep things a bit more tidy/separated from the toolkit config?

Reply:
Oh and then you'll also want to update `package.json` build script to reflect the new path to `copy-pkg-json.js`:

```json
"build": "rollup -c && tsc -p ./tsconfig.json && npm run doc && npm run build:react && ./scripts/copy-pkg-json.js",
```

Reply:
Resolved by a25ce7b